### PR TITLE
chore: Remove local file support for `manifest`  and `defaultCR`

### PR DESCRIPTION
**Pull request by c-pius**
**Description**

Changes proposed in this pull request:

- follow-up to https://github.com/kyma-project/modulectl/issues/53
- `manifest` and `defaultCR` in module config must be links to openly accessible files, not file paths to local files

**Related issue(s)**

### CODE REVIEW

**Comment by c-pius** [comment]
PR on hold. We need to re-discuss the idea of mandating file links. This makes sense for module releases that are put into `module-manifests` repo. The teams first create the module release, than update the config in the repo...

However, is it necessary for module teams to have the local file option during development of the module? We for instance also use the local file option when testing locally or in the E2E test pipeline. Module teams may do the same to quickly push a working state into a local registry. We need to clarify before we merge this.

**Comment by ruanxin** [comment]
basically, module team should not care about ModuleTemplate, `modulectl` is a tool design and used by submission process only, there is no needs for module team to create ModuleTemplate for local development, to verify the functionally of module just need to deploy raw manifest yaml directly, like what we offered in [quick install guide](https://kyma-project.io/#/02-get-started/01-quick-install). Additionally, introducing the option for a local file reference may cause confusion, as it requires the team to have separate configurations for local testing and the submission pipeline, supporting such usage would also increase our effort in terms of documentation and clarification.

Anyway, we can remove this option to simplify the usage first, if later do have the needs to reference local file, we can also support accordingly.

**Review by nesmabadr — APPROVED** [review]

**Pull request merged**
